include/NovelRT/Atom.h

```diff
@@ -1,68 +1,97 @@
 // Copyright © Matt Jones and Contributors. Licensed under the MIT Licence (MIT). See LICENCE.md in the repository root for more information.
 
+#include <atomic>
+#include <cstddef>
+
 #ifndef NOVELRT_ATOM_H
 #define NOVELRT_ATOM_H
 
-#ifndef NOVELRT_H
-#error Please do not include this directly. Use the centralised header (NovelRT.h) instead!
-#endif
```

Comment:
Needed to remove the centralised header warning here? I'm assuming this won't cause any issues down the line? :eyes:

Reply:
I believe the endgame is to remove centralised header warnings from all the includes to allow individual includes, and provide the combined headers as convenience headers

Reply:
Ah okay - I'll resolve this then!